draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/FieldMessage.spec.tsx

```diff
@@ -47,9 +47,9 @@ describe("<FieldMessage />", () => {
     expect(container.querySelector(".reversed")).toBeTruthy()
   })
 
-  it("should render an `error` field message", () => {
+  it("should render a `WarningIcon` with an error", () => {
```

Comment:
```suggestion
  it("should render a warning icon with an error status", () => {
```

Best to avoid technical names within the description.
If we don't want to mention the status in the description, then I feel we should have an extra expectation where we also check there is an error message.

Reply:
Cheers, appreciate the advice 👍   will make that quick ammendment